django/http/response.py

```diff
@@ -397,14 +400,44 @@ def __init__(self, *args, as_attachment=False, filename='', **kwargs):
         self.filename = filename
         super().__init__(*args, **kwargs)
 
+    def _wrap_file_to_stream_close(self, filelike):
+        """
+        Replace the given file-like object's `close()` method with a wrapped
+        version that calls `FileResponse.close()`.
+        """
+        if hasattr(filelike, 'close'):
+            filelike_close = filelike.close
+        else:
+            def filelike_close():
+                pass
```

Comment:
Condense this to the following?
```python
filelike_close = getattr(filelike, 'close', lambda: pass)
```

Reply:
The flake8 check doesn’t allow lambda expressions. It says to use a normal function definition.

Reply:
Really? Surely in this context it should be fine as it is an argument to `getattr()`…

(Personally I don't like that check at it leads to cases like this where we have five lines of code instead of a single readable line.)

Reply:
Okay, you're right. It permits it when it's used e.g. as an argument.